README.md

```diff
@@ -74,43 +74,31 @@ Examples of `Build` resource using the example strategies installed by default.
 ## Try it!
 
 * Get a [Kubernetes](https://kubernetes.io/) cluster and [`kubectl`](https://kubernetes.io/docs/reference/kubectl/overview/) set up to connect to your cluster.
-* Clone this repository from GitHub at the v0.3.0 tag:
+* Install [Tekton](https://cloud.google.com/tekton) `v0.20.1`:
+    ```bash
+    $ kubectl apply -f https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.20.1/release.yaml
+    ```
 
-  ```bash
-  $ git clone --branch v0.3.0 https://github.com/shipwright-io/build.git
-  ...
-  $ cd build/
-  ```
-
-  _Coming soon - install Shipwright Build via kubectl!_
-
-* Install [Tekton](https://cloud.google.com/tekton) by running [hack/install-tekton.sh](hack/install-tekton.sh), it installs v0.20.1.
+* Install Shipwright build controller `v0.3.0`:
 
   ```bash
-  $ hack/install-tekton.sh
+  $ kubectl apply --filename https://github.com/shipwright-io/build/releases/download/v0.3.0/release.yaml
   ```
 
-* Install Shipwright and sample strategies via `make`:
+* Install Buildpacks sample build strategy:
 
   ```bash
-  $ make install
-  ```
-
-* Add a push secret to your container image repository, such as one on Docker Hub or quay.io:
-
-  ```yaml
-  $ kubectl create secret generic push-secret \
-  --from-file=.dockerconfigjson=$HOME/.docker/config.json \
-  --type=kubernetes.io/dockerconfigjson
+  $ kubectl apply --filename https://raw.githubusercontent.com/shipwright-io/build/v0.3.0/samples/buildstrategy/buildpacks-v3/buildstrategy_buildpacks-v3_cr.yaml
   ```
+  More sample build strategies can be found in [sample buildstrategy folder](https://github.com/shipwright-io/build/tree/master/samples/buildstrategy)
 
 * Create a [Cloud Native Buildpacks](samples/build/build_buildpacks_v3_cr.yaml) build, replacing
   `<MY_REGISTRY>/<MY_USERNAME>/<MY_REPO>` with the registry hostname, username, and repository your
-  cluster has access to and that you have permission to push images to.
+  cluster has access to and that you have permission to push images to by creating or using the secret `push-secret`:
 
   ```bash
   $ kubectl apply -f - <<EOF
-  apiVersion: shipwright.io/v1alpha1
+  apiVersion: build.dev/v1alpha1
```

Comment:
Isnt this wrong? in theory this PR goes to master where we have already the new domain. We should not change this.

Reply:
Because the `Try It` is not for the latest commit, it uses the release of Shipwright build controller `v0.3.0`
At that time, the API is still `build.dev/v1alpha1`

Reply:
sure, but you cannot change master README to point to an old domain. You either need to temporary reference here a [sample](https://github.com/shipwright-io/build/blob/v0.3.0/samples/build/build_kaniko_cr.yaml) from the old release or similar an once we ship the v0.4.0 you can enhance the documentation.

Reply:
Yes, remove the yaml file code and point to the sample url directly.